src/VkBootstrap.cpp

```diff
@@ -819,6 +835,14 @@ InstanceBuilder& InstanceBuilder::set_minimum_instance_version(uint32_t major, u
     info.minimum_instance_version = VKB_MAKE_VK_VERSION(0, major, minor, patch);
     return *this;
 }
+InstanceBuilder& InstanceBuilder::force_api_version(uint32_t major, uint32_t minor) {
```

Comment:
I don't have any meaningful suggestions so I'll bikeshed the name

What about `set_api_version`? `force_api_version` makes so... forceful

Reply:
This use case is kinda "advanced" API usage, so a scary name helps ward away unsuspecting users.

Reply:
I went ahead and removed the function entirely - having a function which did zero checking didn't sit right with me.